packages/connectivity/connectivity/android/src/main/java/io/flutter/plugins/connectivity/Connectivity.java

```diff
@@ -103,4 +118,38 @@ private String getNetworkTypeLegacy() {
   public ConnectivityManager getConnectivityManager() {
     return connectivityManager;
   }
+
+  private Boolean checkRequirement() {
```

Comment:
nit: Can we name this more descriptively? Maybe something like `checkPermissions`?

Reply:
PR updated. Thank you